src/lib/components/LandingPage.svelte

```
@@ -88,11 +98,47 @@
 				<div class="px-5 py-4">
 					<div class="text-foreground prose prose-sm text-[13px]/relaxed">
 						{@html testDetails.start_instructions}
```

ESLint

[error] `{`@html`}` can lead to XSS attack.

(svelte/no-at-html-tags)